makefile

```diff
@@ -1,16 +1,18 @@
 CC=gcc
-CFLAGS= -Wall -Wextra
+# CFLAGS= -Wall -Wextra -Werror
```

Comment:
why ?

Reply:
So you have something to comment! But I accept.